DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/Index.cshtml

```diff
@@ -5,6 +5,7 @@
 @model CentreAdministratorsViewModel
 
 <link rel="stylesheet" href="@Url.Content("~/css/shared/searchableElements.css")" asp-append-version="true">
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/centreAdministrators.css")" asp-append-version="true">

```

Comment:
Could we import the searchableElements css in the centreAdministrators sass file?